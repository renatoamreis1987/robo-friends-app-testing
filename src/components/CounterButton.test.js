import { shallow } from 'enzyme';
import React from 'react'
import CounterButton from './CounterButton';
import toJson from 'enzyme-to-json';



it('expect to render Card Component', () => {
    const mockColor = 'red'
    expect(toJson(<CounterButton color={mockColor} />)).toMatchSnapshot()
})
