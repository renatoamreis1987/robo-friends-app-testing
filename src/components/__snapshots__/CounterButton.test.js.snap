// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Component 1`] = `
<button
  color="red"
  onClick={[Function]}
>
  Count: 
  0
</button>
`;
